helm-charts/splunk-otel-collector/templates/clusterRole.yaml

```diff
@@ -87,3 +97,48 @@ rules:
   - get
   - list
   - watch
+---
+{{- if eq .Values.distro "openshift" }}
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: {{ template "splunk-otel-collector.serviceAccountName" . }}
+  labels:
+    app: {{ template "splunk-otel-collector.name" . }}
+    chart: {{ template "splunk-otel-collector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+users:
+- system:serviceaccount:{{ .Release.Namespace }}:{{ template "splunk-otel-collector.serviceAccountName" . }}
+priority: 10
+allowHostNetwork: true 
+allowHostPorts: true
+allowHostPID: true
+allowHostDirVolumePlugin: true
+allowHostIPC: false
```

Comment:
Is the access to IPC namespace required? I may be wrong but I think it's not needed for default chart functionality, not sure about openshift

Reply:
It is set to `false`. Are you saying it should be true?

Reply:
Oh my bad. I misread the boolean value :)